mindmeld/converter/test.py

```diff
@@ -0,0 +1,7 @@
+from mindmeld.converter.dialogflow import DialogflowConverter
```

Comment:
we should consolidate this into `test_df_converter` to make sure that this function is always invoked before we can load and test the NLP app

Reply:
Deleted this file since the sample app has changed.

Reply:
should this file be deleted?